angular.json

```diff
@@ -138,7 +138,7 @@
                   "with": "apps/api/src/environments/environment.prod.ts"
                 }
               ],
-              "externalDependencies": "none"
+              "externalDependencies": "all"
```

Comment:
what is this for?

Reply:
TL;DR: To be able to do `ng build api --prod`

Nx tries to bundle NestJS with Webpack (just like it does with React) but it won't work. [Here's why](https://github.com/nestjs/nest/issues/1706#issuecomment-474657479). So we're telling Nx to not bundle the NestJS app (which doesn't really help a lot given that we don't need bundling in the server). If all the dependencies are "external" then Webpack will leave alone all the occurrences of "require" instead of trying to bundle them.

Reply:
alright, thanks for clearing this up :)